content/intro-curriculum/C_Clean.Rmd

```diff
@@ -3,6 +3,7 @@ title: "C. Clean"
 author: "Jeffrey W. Hollister, Luke Winslow, Lindsay Carr"
 date: "9999-01-08"
 slug: "Clean"
+draft: "FALSE"
```

Comment:
Remove this?

Reply:
I think we want this. Without it, when you click "knit" it will automatically add `draft: true` to the `.md`. With it, we have declared that this is no longer in draft mode and each small change does not need a huge review.